foreman.te

```diff
@@ -495,6 +495,9 @@ optional_policy(`
 files_search_pids(httpd_t)
 stream_connect_pattern(httpd_t, foreman_var_run_t, foreman_var_run_t, foreman_rails_t)
 
+# Allow Apache access to static assets
+allow httpd_t foreman_lib_t:file { getattr map open read };
```

Comment:
It also allows reading everything in `/var/lib/foreman`. Now that we no longer store the (potential) sqlite DB there, I guess it's fairly safe, though I can imagine we'd store more secrets there in the future.

I've been thinking about introducing its own type that both Foreman and Apache are able to read. It may also be that you must set `config.public_file_server.enabled = false` in the settings and then it no longer tries to read them.